style.css

```diff
@@ -0,0 +1,386 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	font-size: 10px;
+	font-family: Ubuntu, Sans-serif;
+	color: white;
+	font-weight: bold;
+}
+br {
+	display:none;
+}
+body {
+	background-color: #C06C84 ;
+	width: 100%;
+	font-family: "Idim";
+}
+nav {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	height: 50px;
+	position: fixed;
+	padding: 15px;
+	background-color: #C06C84;
+}
+
+footer {
+	text-align: center;
+	background-color: #355C7D;
+	padding-bottom: 30px;
+}
+
+a:visited {
+	color: inherit;
+}
+
+
+
+.nav-left-container {
+	display: flex;
+	justify-content: flex-start;
+	gap: 30px;
+	color: white;
+}
+
+.nav-left-container a {
+	text-decoration: none;
+	font-size: 2rem;
+	color: white;
+}
+
+nav > a {
+	font-size: 2rem;
+	text-decoration:none;
+}
```

Comment:
the `>` selector means direct child, so only applies to `a` directly inside `nav`. there is no such elements, therefore this code does not apply to anything

Reply:
whoops. there was another selector that was taking care of this. thanks. done